src/js/components/Avatar/stories/CustomThemed/Stacked.tsx

```diff
@@ -76,10 +75,10 @@ export const Stacked = () => {
         <GroupedGravatarCentered />
         <GroupedGravatarRTL />
       </Box>
-    </Grommet>
+    </Box>
   );
 };
 
 export default {
-  title: 'Visualizations/Avatar/Stacked',
+  title: 'Visualizations/Avatar/Custom Themed/Stacked',
```

Comment:
This story shouldn't go in the custom themed folder since it doesn't use a custom theme. Instead it looks like we need to change the colors like in the Basic story since the colors for some of the Avatar's are not defined in the hpe theme.